compiler/tests/fail/stack_allocation/x86-64/reg_var_expected.jazz

```diff
@@ -0,0 +1,9 @@
+export fn main () -> reg u64 {
```

Comment:
Why knowing where the variable is defined is relevant for the error

Reply:
cf. https://github.com/jasmin-lang/jasmin/pull/1124